docs/src/main/asciidoc/grpc-getting-started.adoc

```diff
@@ -175,9 +176,9 @@ public class HelloService implement Greeter {  <2>
     }
 }
 ----
```

Comment:
I _just_ found that the snippet of code above can't compile, because it uses `implement` instead of `implements` :-)

Not your fault of course.

Reply:
Good catch! Whose fault is it if not mine... git blames me that I'm the author of the original snippet too ;-)

Reply:
Umm okay :-)